src/python/src/setup.py

```diff
@@ -80,7 +81,15 @@
     'grpc.framework': 'grpc/framework',
 }
 
-_core.setup(
-    name='grpc-2015', version='0.4.0',
-    ext_modules=[_EXTENSION_MODULE], packages=list(_PACKAGES),
-    package_dir=_PACKAGE_DIRECTORIES)
+setuptools.setup(
+    name='grpcio',
+    version='0.4.0a3',
```

Comment:
Yes - after an initial snafu wherein I submitted a 0.4.0 package and deleted it from PyPI, I could not resubmit 0.4.0 due to it having the same file name as a file that existed at some point in the past.